### DISCUSSION TITLE
Air for Mac OS 33.1.1.932 - disappeared?

### DISCUSSION THREAD

**Discussion by tomroyal** [airsdk/Adobe-Runtime-Support#2707] [Q&A] [answered]
Hi

I'm trying to debug an issue with an Air application on MacOS. The last known good configuration had it running via Air Runtime v33.1.1.932; this was uninstalled in order to attempt a clean install.

Now I find that v33.1.1.932 appears to have disappeared. Our options are:

50.2.1.1 - which installs perfectly, but the application does not recognise it and requests a "more recent" version of Air
33.1.1.744 - which hangs on install

I leaned via another thread here that 33.1.1.932 introduced a change to the WebView - did that become the 50.x branch? And in the meantime, does anyone know a (safe) source for a v33.1.1.932 Mac installer?

Thanks in advance - Tom

**Comment by ajwfrost** [comment]
Hi

Concerning if you're finding 33.1.1.744 hanging on install - can you please confirm what sort of Mac and what version of macOS you have there?

The 744 version is the last one that has the old webkit code, so yes the HTML support changed after that. And yes the 50.x continues on from that but with other AS3 updates. If the app doesn't recognise this, it may be because there is a dependency on the HTML stuff although if it previously worked with 33.1.1.932 then this shouldn't be the issue! Do you mean, when you click on the .air application or are installing a .pkg file, then it says it can't find AIR or needs a later version?

I don't know whether 33.1.1.932 would actually help, as it is more likely that an update in macOS security is causing problems with the shared runtime approach, but I'll dig out that version and drop a download link below..

thanks

**Reply by ajwfrost** [reply, marked as answer]
https://transfer.harman.com/link/j4l05EIb31ChrCVJchlgEu/download/AdobeAIR_33.1.1.932.dmg for AIR shared runtime 33.1.1.932 on macOS. Link expires 12 July 2023.
Checksum: `b6b9bd83b3bff541481aff18f7b2238c536b53d4a29d0cdfb749ae93c3a8095b`

**Comment by tomroyal** [comment]
Hi Andrew - thanks very much for the quick response, and for the installer link.

The Mac is an M1 MacBook Pro running OSX 12.3. I'm getting hands-on access to it tomorrow - will try this installer, and if that fails will check out exactly what happens when we try the 50.x branch. Will report back either way.

Thanks again,

Tom

**Comment by tomroyal** [comment]
OK, so: this installer runs and completes. Version 33.1.1.932 is installed. The Air application in question also installs. 

On launch on OSX 12.3 the Harman Air splash is shown, but then the application hangs. This was the original problematic behaviour.

I then updated OSX to 12.6.7. Air 33.1.1.932 remains installed, and the Air application also runs successfully.

It's hard for me to discern from this exactly where the problem manifested - but at least we're up and running again. 

I do also have the log from the failed installation of .744 on OSX 12.3 - please let me know if you'd like to see that.

Thanks again

Tom
